django/core/management/commands/squashmigrations.py

```diff
@@ -32,6 +32,10 @@ def add_arguments(self, parser):
             '--noinput', '--no-input', action='store_false', dest='interactive', default=True,
             help='Tells Django to NOT prompt the user for input of any kind.',
         )
+        parser.add_argument(
+            '--finalname', dest='final_name', default=None, nargs=1,
+            help='Use this name for the resulting migration',
+        )
```

Comment:
--file-name or --filename? Something like --squashed-name might be more intuitive.